.ahoy.yml

```diff
@@ -31,7 +31,9 @@ commands:
     usage: Build and start Docker containers.
     cmd: |
       docker-compose up -d "$@"
-      ahoy cli "dockerize -wait tcp://ckan:3000 -timeout 1m"
+      sleep 30
```

Comment:
you can download the 'dockerize' binary and run it, its still a good thing to use just in case things take too long to boot. or the opp, its done in 2sec but you still waited 30.

Reply:
We do have `dockerize`, but for debugging purposes, I wanted to grab the docker-compose logs after startup had had a chance to run (which turned out to be a good idea, they were very useful). I tried to get the logs only on dockerize timeout, but it didn't seem to work. I don't really want to drop it entirely, but now that it's working, I could maybe lower the enforced sleep to 10 seconds?